### PR TITLE
Renamed IPC to IPCRenderer in plugins

### PR DESCRIPTION
**Pull request by Mingling94**
*Fixed some IPC listeners to electron v0.35.0 standard
*Moved around some code in Files to be closer to where they're relevant

### CODE REVIEW

**Comment by Mingling94** [comment]
@DavidVorick @lukechampine this should be pretty uncontroversial

File: plugins/Files/js/loaders.js [hunk 1]
```diff
@@ -72,6 +72,10 @@ addResultListener('file-loaded', function(result) {
 	update();
 });
 
+// Checks whether a path starts with or contains a hidden file or a folder.
+function isUnixHiddenPath(path) {
+	return (/(^|\/)\.[^\/\.]/g).test(path);
+}
```
Comment (DavidVorick) [inline plugins/Files/js/loaders.js]: This probably belongs in a separate PR

File: plugins/Files/js/lifecycle.js [hunk 2]
```diff
@@ -20,6 +20,16 @@ addResultListener('update-list', function(result) {
...
 
+// Convert to Siacoin
+// TODO: Enable commas for large numbers
+function convertSiacoin(hastings) {
+	// TODO: JS automatically loses precision when taking numbers from the API.
+	// This deals with that imperfectly
+	var number = new BigNumber(Math.round(hastings).toString());
+	var ConversionFactor = new BigNumber(10).pow(24);
+	return number.dividedBy(ConversionFactor).round();
+}
```
Comment (DavidVorick) [inline plugins/Files/js/lifecycle.js]: this probably belongs in a separate PR
